### DISCUSSION TITLE
I mistakenly transferred my 115 usdt(btc20) from my trust wallet to usdt(btc20) contract address

### DISCUSSION THREAD

**Discussion by graceten10** [blockscout/blockscout#11004] [Support]
I mistakenly transferred my 115 usdt(btc20) from my trust wallet to usdt(btc20) contract address . Please how can I get my money back? Here is my receiving address 0x2fa08CdcDadb10A5C30EE94B9A80525C1F3be22C

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
